superset/migrations/versions/2025-11-18_12-00_e8f4a3b2c1d0_add_mcp_api_keys.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add api keys
+
+Revision ID: e8f4a3b2c1d0
+Revises: x2s8ocx6rto6
+Create Date: 2025-11-18 12:00:00.000000
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e8f4a3b2c1d0"
+down_revision = "x2s8ocx6rto6"
+
+
+def upgrade():
+    """Create ab_api_key table for API key authentication."""
+    op.create_table(
+        "ab_api_key",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.String(length=256), nullable=False),
+        sa.Column("key_hash", sa.String(length=128), nullable=False),
+        sa.Column("key_prefix", sa.String(length=8), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("workspace_name", sa.String(length=256), nullable=False),
```

Comment:
This seems Preset specific, as I don't believe vanilla Superset has workspaces. So I think this should probably be removed?